contrib/rustypaste/files/rustypaste.default

```diff
@@ -0,0 +1,4 @@
+# path to the config file
```

Comment:
name this config.toml and then you don't have to rename it with name= below

Reply:
yes that one is merely to pass around the env var through dinit to the process. Otherwise rustypaste wouldn't know where to look for its config file.
the auth variable is just for convenience though